test/library/packages/Itertools/cycle/check_indexable.chpl

```diff
@@ -0,0 +1,11 @@
+use Itertools;
+
+var A: [1..10] int;
+forall (id, el) in zip(1..10, cycle((1, 2, 3, 4))) do A[id] = el;
+writeln(A);
+forall (id, el) in zip(1..10, cycle('ABC')) do A[id] = ascii(el):int;
```

Comment:
`ascii` has since been deprecated on master -- so we need to update this:

```suggestion
forall (id, el) in zip(1..10, cycle('ABC')) do A[id] =el.byte(1):int;
```

Reply:
Done.
Just one doubt, why is it needed to typecast the result to `int`? As per my understanding, `byte()` returns an `int` only?

Reply:
[byte()](https://chapel-lang.org/docs/master/builtins/String.html#String.string.byte) returns a `uint(8)`, so the cast is necessary.